apps/web/components/article-with-multiple-authors/tailwind.tsx

```diff
@@ -11,71 +11,74 @@ import { Layout } from "../_components/layout";
 
 export const component = (
   <Row>
-    <Hr className="my-4 border-gray-300" />
-    <Section className="flex">
+    <Hr className="mb-0 mt-[16px] border-gray-300" />
+    <Section>
       {[
         {
-          name: "Zeno Rocha",
+          name: "Steve Jobs",
           title: "Co-Founder & CEO",
-          imgSrc: "/static/zeno-rocha.jpeg",
+          imgSrc: "/static/steve-jobs.jpg",
           showDivider: true,
         },
         {
-          name: "Bu Kinoshita",
+          name: "Steve Wozniak",
           title: "Co-Founder & CTO",
-          imgSrc: "/static/bu-kinoshita.jpeg",
+          imgSrc: "/static/steve-wozniak.jpg",
           showDivider: false,
         },
       ].map((author, index) => (
-        <Section
-          className={`w-1/2 pl-${author.showDivider ? "0" : "4"} ${
-            author.showDivider ? "border-r border-gray-300" : ""
-          }`}
-          key={index}
-        >
-          <Section className="mt-1.5 inline-block max-h-12 max-w-12 text-left">
-            <Img
-              alt={author.name}
-              className="block h-12 w-12 rounded-full object-cover object-center"
-              src={author.imgSrc}
-            />
-          </Section>
-          <Section className="ml-4 inline-block max-w-[120px] text-left align-top">
-            <Heading
-              as="h3"
-              className="m-0 text-sm font-medium leading-5 text-gray-900"
-            >
-              {author.name}
-            </Heading>
-            <Text className="m-0 text-xs font-medium leading-4 text-gray-500">
-              {author.title}
-            </Text>
-            <Section className="mt-1">
-              <Link className="h-3 w-3 text-gray-400" href="#">
-                <svg
-                  fill="currentColor"
-                  height="12"
-                  viewBox="0 0 16 16"
-                  width="12"
-                  xmlns="http://www.w3.org/2000/svg"
-                >
-                  <path d="M12.6.75h2.454l-5.36 6.142L16 15.25h-4.937l-3.867-5.07-4.425 5.07H.316l5.733-6.57L0 .75h5.063l3.495 4.633L12.601.75Zm-.86 13.028h1.36L4.323 2.145H2.865z" />
-                </svg>
-              </Link>
-              <Link className="ml-2 h-3 w-3 text-gray-400" href="#">
-                <svg
```

Comment:
SVGs don't have the best support, we should have an image for this like we do for other icons